packages/webpack-utilities/just.config.ts

```diff
@@ -1,4 +1,30 @@
-import { preset, task } from '@fluentui/scripts';
+import * as path from 'path';
+import * as fs from 'fs';
+import { preset, task, series, TaskFunction } from '@fluentui/scripts';
 
 preset();
-task('build', 'build:node-lib').cached();
+
+/**
+ * Task that renames a folder
+ *
+ * during build task simplification every cjs module output will be emitted under `lib-commonjs` folder.
+ * because this package violates that rule and it docs specifies `lib` as public api in order
+ * to not introduce breaking change into this legacy package we need this workaround.
+ */
+function renameFolder(options: { from: string; to: string }): TaskFunction {
+  return done => {
```

Comment:
if the command is sync why does the task need to use an async `done()` callback ?

Reply:
I was just following source code of just-task as there is no official documentation AFAIK. that's the definition of `TaskFunction`